File: apps/feedback-service-admin/src/api/feedback/FeedbackUpdateInput.ts
import { ReviewUpdateManyWithoutFeedbacksInput } from "./ReviewUpdateManyWithoutFeedbacksInput";

export type FeedbackUpdateInput = {
  booking?: string | null;
  comments?: string | null;
  rating?: number | null;
  reviews?: ReviewUpdateManyWithoutFeedbacksInput;
};

File: apps/billing-service/src/APIs/Feedback/Dtos/FeedbackUpdateInput.cs
namespace BillingService.APIs.Dtos;

public class FeedbackUpdateInput
{
    public string? Comments { get; set; }

    public DateTime? CreatedAt { get; set; }

    public string? Customer { get; set; }

    public string? Id { get; set; }

    public int? Rating { get; set; }

    public DateTime? UpdatedAt { get; set; }
}
